pdns/logger.hh

```diff
@@ -116,7 +116,7 @@ private:
   static pthread_key_t s_loggerKey;
 };
 
-extern Logger &theL(const string &pname="");
+extern Logger s_log;
```

Comment:
So, hum, why `s_log`? This variable is not static so perhaps `g_log` would make more sense?

Reply:
oh, indeed... This is me sucking at C++ because the `static` keyword gave all kinds of issues.. I'll se eif I can fit it with a fresh brain

Reply:
right, i incorrectly suggested s_..., sorry!